TinyMCE.Blazor/Editor.razor

```diff
@@ -95,6 +103,7 @@
       _value = content;
       ValueChanged.InvokeAsync(content);
```

Comment:
The `TriggerValidation` should go inside the callback to ensure we have the updated value in case of long contents
```suggestion
      ValueChanged.InvokeAsync(content);
      TriggerValidation();
```

Reply:
Just this tiny change and we should be good

Reply:
It should indeed. Also UpdateText?